includes/Dashboard/Templates/Settings.php

```diff
@@ -647,4 +735,99 @@ function get_dokan_categories() {
 
         return apply_filters( 'dokan_category', $dokan_category );
     }
+
+    /**
+     * Get proper heading for payments of vendor dashboard payment settings
+     *
+     * @since DOKAN_SINCE
+     *
+     * @param string $slug
+     * @param string $heading
+     *
+     * @return string
+     */
+    private function get_payment_heading( $slug, $heading ) {
+        switch ( $slug ) {
+            case 'bank':
+            case 'bank/edit':
+                $heading = __( 'Bank Account Settings', 'dokan-lite' );
+                break;
+
+            case 'paypal':
+            case 'paypal/edit':
+                $heading = __( 'Paypal Settings', 'dokan-lite' );
+                break;
+        }
+
+        /**
+         * To allow new payment extension give their own heading
+         *
+         * @since DOKAN_SINCE
+         *
+         * @param string $heading previous heading
+         */
+        $heading = apply_filters( "dokan_withdraw_method_settings_title", $heading, $slug );
+
+        return $heading;
+    }
+
+    /**
+     * Get unused payment methods
+     *
+     * @since DOKAN_SINCE
+     *
+     * @param array $all_methods          All methods
+     * @param array $used_methods         Used Methods
+     * @param array $method_key_mismatchs The list of methods which has different key and name but the name is substring of key
+     *
+     * @return array
+     */
+    private function get_used_and_unused_payment_method_keys( $all_methods, $used_methods, $method_key_mismatchs ) {
```

Comment:
Update the method name here. And in the description, describe what this method is doing. Also, the description and method name doesn't match here.